authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -87,22 +91,42 @@ public RangerAuthorizationPlugin(String catalogProvider, Map<String, String> con
     RangerHelper.check(password != null, "Ranger password is required");
     RangerHelper.check(rangerServiceName != null, "Ranger service name is required");
     rangerClient = new RangerClientExtend(rangerUrl, authType, rangerAdminName, password);
-    rangerHelper = new RangerHelper(this, catalogProvider);
+
+    // Initialize privilegesMapping and ownerPrivileges
+    ownerPrivileges = ownerMappingRule();
+    privilegesMapping = privilegesMappingRule();
+    policyResourceDefines = policyResourceDefinesRule();
+
+    rangerHelper =
+        new RangerHelper(
+            rangerClient,
+            rangerAdminName,
+            rangerServiceName,
+            privilegesMapping,
+            ownerPrivileges,
+            policyResourceDefines);
+  }
+
+  public final Map<Privilege.Name, Set<RangerPrivilege>> getPrivilegesMapping() {
+    return privilegesMapping;
+  }
+
+  public final Set<RangerPrivilege> getOwnerPrivileges() {
+    return ownerPrivileges;
+  }
+
+  public final List<String> getPolicyResourceDefines() {
+    return policyResourceDefines;
```

Comment:
Also do you still need these methods? I think you can directly call `privilegesMappingRule`, `ownerMappingRule`, etc.